From 5c29bb9fbab209c4aa7863fbb0b3ae6e251d407a Mon Sep 17 00:00:00 2001
From: Nancy Chen <Nancy.Chen@freescale.com>
Date: Thu, 31 Jul 2008 10:55:20 -0500
Subject: [PATCH] ENGR00085977 Voltage for 266MHz is not 1.25V

Voltage for 266MHz is not 1.25V while DVFS is enabled on MX31ADS and MX32ADS

Signed-off-by: Nancy Chen <Nancy.Chen@freescale.com>
---
 arch/arm/mach-mx3/dvfs_v2.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx3/dvfs_v2.c b/arch/arm/mach-mx3/dvfs_v2.c
index fa40a24..152caf0 100644
--- a/arch/arm/mach-mx3/dvfs_v2.c
+++ b/arch/arm/mach-mx3/dvfs_v2.c
@@ -324,6 +324,9 @@ void pmic_voltage_init(void)
 {
 	t_regulator_voltage volt;
 
+	/* Enable 4 mc13783 output voltages */
+	pmic_write_reg(REG_ARBITRATION_SWITCHERS, (1 << 5), (1 << 5));
+
 	/* Set mc13783 DVS speed 25mV each 4us */
 	pmic_write_reg(REG_SWITCHERS_4, (0 << 6), (3 << 6));
 
-- 
1.5.4.4

